### PR TITLE
remove special handling for superusers

### PR DESCRIPTION
**Pull request by indirectlylit**
should treat superuser like other users, with only anonymous users still being "special"

not included in this PR: updating UX and nomenclature (device owners, permission stars, etc)

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 1]
```diff
@@ -182,17 +182,11 @@
         }
       },
       showNextBtn() {
-        return this.progress >= 1 && this.content && this.nextContentLink;
+        return this.sessionProgress >= 1 && this.content && this.nextContentLink;
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue]: This is incorrect. Should be summaryProgress, not sessionProgress.

Also, this probably should have been conditionalized for anonymous users as well?

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 2]
```diff
@@ -182,17 +182,11 @@
...
       },
       recommendedText() {
         return this.$tr('recommended');
       },
-      progress() {
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue]: I would say reinstate this, and just check that user is logged in.

**Comment by indirectlylit** [comment]
cool.

I accidentally deleted one of your comments, but the gist was:

> reinstate the `progress()` computed prop, and switch it based on logged-in state

**Comment by indirectlylit** [comment]
@rtibbles - updated

**Pull request merged**